keyservice/server.go

```diff
@@ -13,10 +14,30 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+func NewServer(prompt bool) *Server {
+	return &Server{prompt, kmsKeyProvider{}};
+}
+
 // Server is a key service server that uses SOPS MasterKeys to fulfill requests
 type Server struct {
 	// Prompt indicates whether the server should prompt before decrypting or encrypting data
-	Prompt bool
+	Prompt               bool
+	kmsMasterKeyProvider kmsMasterKey
+}
+
+type kmsMasterKey interface {
+	NewKMSMasterKey(svcKey *KmsKey, ctx map[string]*string) keys.MasterKey
+}
+
+type kmsKeyProvider struct{}
+
+func (kmsKeyProvider) NewKMSMasterKey(svcKey *KmsKey, ctx map[string]*string) keys.MasterKey {
+	return &kms.MasterKey{
+		Arn:               svcKey.Arn,
+		Role:              svcKey.Role,
+		EncryptionContext: ctx,
+		AwsProfile:        svcKey.AwsProfile,
+	};
```

Comment:
Please run `go fmt`, will remove this semi-colon.

Reply:
My bad! I'll fix the formatting with `go fmt` later today.